### ISSUE TITLE
forter.com FP?

### ISSUE THREAD

**Issue by spirillen** [jawz101/MobileAdTrackers#28]
Hi @jawz101 

I'm wondering why you have `*.forter.com` listed in your hosts file. 

A view on there website, they are a fraud prevention system. I know I know, the is easily as falsum as Chrome should be safe and secure :-1: and that's why I'm asking you :smiley:

**Comment by jawz101** [comment]
I think I'll remove it.  There are many that need to be pruned from some bigger merges I did from other lists and those entries originated from a huge import based on the top 1 million OpenDNS queries.  Needless to say it was pretty aggressive attempt to find additional blocks.  It looks like that company does have a mobile sdk which I would be inclined to block if it didn't cause problems but since I haven't seen it myself I will remove it.

If I do see it I may try to selectively pick its subdomains to block that do not cause breakage.  Simply because the idea of fraud prevention could inherently be trying to see how you use your device.

**Comment by spirillen** [comment]
wow OpenDNS top 1mill for initial load, that's heavy :face_with_head_bandage: But I like your approach, better safe than sorry. and it can't be all fu.. up if anything haven't been reported as broken :+1: 

But don't worry, I to have some FP that I haven't found yet. You'll need feedback fro mother that's might have been inclined by any of your submissions. 

Keep up the good work :+1:

**Comment by jawz101** [comment]
Yeah that was a dumb idea at the time on my part.  It wasn't the initial load but something I did a couple of years ago to find additional domains.  I scrubbed the 1 million down to about 24,000 new entries and added that chunk but it ended up being some IoT stuff, internal company domain names, and other random stuff so I reverted most of it over time.

Ideally, my list (or now the Adaway list) should be maybe 4-5000 entries.  Anything else is browser-based domain names which I want to keep out of the list unless they show up in mobile app analytic/tracking libraries.

**Comment by spirillen** [comment]
That's ok

**Comment by jawz101** [comment]
I just had an idea to use that Cisco/OpenDNS/Umbrella 1 million list to potentially remove entries.  If the entries don't show up as one of the top 1 million most popular DNS lookups it possibly isn't worth having on this list (or Adaway).
http://s3-us-west-1.amazonaws.com/umbrella-static/index.html

I can also check it against the Majestic 1 million as well.
https://majestic.com/reports/majestic-million

**Comment by spirillen** [comment]
That can be a god point if you are going after a smaller list, but it would be very narrow, to a limited number of users.

The perfect would be, to have a server that was scraping the top 10.000.000 for common DNS records and scripts. But that would require a bit more equipment and a good coder.

I have also been tumbling with another idea....

We are quit a few people contributing to a verity of different repos etc. why is it we don't have a common report forum, from where every list owner had a central source, to find what suited the individual lists purpose. For now I'm trying to start it up in my [Matrix](https://gitlab.com/my-privacy-dns/matrix/matrix/issues) dummy copy (https://github.com/mypdns/matrix) by the way i have build up the issue templates.

**Comment by jawz101** [comment]
I kinda liked the system FireHOL has in place for IP blocklist analysis [here](http://iplists.firehol.org/).  I asked the developer a while ago if he would consider doing the same for domain name lists but he's also involved in [another, bigger project](https://www.netdata.cloud/).

**Comment by spirillen** [comment]
Yeah I know. It kind of the same I'm thinking about. The start structure is there :star: 

And then the second part..... why rebuild something that's actually already is developed as a nice forum (issues) with a strong search interface :mag:
